lightningd/opt_time.c

```diff
@@ -4,6 +4,7 @@
 #include <errno.h>
 #include <inttypes.h>
 #include <stdio.h>
+#include <assert.h>
```

Comment:
Not sorted.

Reply:
Doh! Will fix. Thanks for the make check-source command-line tip.

Reply:
`flake8` is yet another Python dependency we have pulled in.  It seems to be available on most distributions as the `flake8` package.  I myself have not managed to get `pylightning` (and hence the `tests/test_lightningd.py` test) running on my hacking machines and am dependent on the Travis CI setup on my account for that.